opal/mca/common/cuda/common_cuda.c

```diff
@@ -126,6 +126,9 @@ static CUstream ipcStream = NULL;
 static CUstream dtohStream = NULL;
 static CUstream htodStream = NULL;
 static CUstream memcpyStream = NULL;
+#if OPAL_CUDA_GPU_MEM_WAR
+static int mca_common_cuda_gpu_mem_check_war = 1;
```

Comment:
Shouldn't this *not* be protected by an `#if`, and instead just set the C value to the macro?

```
static int mca_common_cuda_gpu_mem_check_war = OPAL_CUDA_GPU_MEM_WAR;
```

That means that this value is *always* available, but its default value is dependent on the version found in `cuda.h`.